documental/models.py

```diff
@@ -0,0 +1,415 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from documental.utils import (
+    diretorio_mapas_uso_solo,
+    diretorio_ti
+)
+
+
+class Action(models.Model):
+    """Action model data for documental model."""
+
+    # id = models.IntegerField(
+    #     _('Primary key'),
+    #     unique=True,
+    #     primary_key=True,
+    # )
+
+    no_acao = models.CharField(
+        _('Action name'),
+        max_length=255,
+        unique=True,
+    )
+
+    dt_cadastro = models.DateTimeField(
+        _('Registration date'),
+        null=True,
+        blank=True,
+    )
+    class Meta:
```

Comment:
Missing blankspace.

Are you using pep8 formatter?

Reply:
Removed,